PremiereClipboard/Program.cs

```diff
@@ -10,33 +10,40 @@ namespace PremiereClipboard
 {
     class Program
     {
-        static string trackFormat = "PProAE/Exchange/TrackItem";
+        const string trackFormat = "PProAE/Exchange/TrackItem";
+        static bool overwriteProtection = true;
```

Comment:
static things should start with a capital letter.

soruce: https://docs.microsoft.com/en-us/previous-versions/dotnet/netframework-1.1/d53b55ey%28v%3dvs.71%29